pkgs/tools/misc/coreboot-utils/default.nix

```
@@ -84,6 +84,8 @@ let
     amdfwtool = generic {
       pname = "amdfwtool";
       meta.description = "Create AMD firmware combination";
+      buildInputs = [ openssl ];
+      nativeBuildInputs = [ pkgconfig ];
```

```suggestion
      nativeBuildInputs = [ pkg-config ];
```
that was an alias unless you mean the python package

---

Thanks!